### PR TITLE
CI for container images

### PR DESCRIPTION
**Pull request by rigtorp**
Using Github actions because it's easier for me to work with.

**Changes**

Build and push container images nightly and when commits are pushed to release tags (`v*`).

**Issues**

### CODE REVIEW

**Comment by rigtorp** [comment]
@Artiume I'm not able to reliably build the container images: https://github.com/rigtorp/jellyfin/actions/runs/98615511

**Comment by Artiume** [comment]
Pull this. I added ogg as both audio and video mime type and it did not like it. 

https://github.com/jellyfin/jellyfin/pull/3066

**Comment by rigtorp** [comment]
@Artiume No the issue seems to be web UI does not build. Previously I had issue with JF PGP keys

File: Dockerfile.arm [hunk 1]
```diff
@@ -35,35 +34,35 @@ ARG APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=DontWarn
...
+   jellyfin-ffmpeg \
+   libssl-dev \
+   libfontconfig1 \
+   libfreetype6 \
+   libomxil-bellagio0 \
+   libomxil-bellagio-bin \
+   libraspberrypi0 \
+   vainfo \
+   libva2 \
+   openssl \
```
Comment (Artiume) [inline Dockerfile.arm:56]: did this build? iirc i had to use `libssl-dev`

Comment (rigtorp) [reply Dockerfile.arm:56]: Yes I finally have automatic builds working: https://hub.docker.com/repository/registry-1.docker.io/rigtorp/jellyfin/tags?page=1

**Comment by rigtorp** [comment]
@dkanada @joshuaboniface Take a look at this. This PR will allow for nightly and automatic release builds of container images.

**Comment by dkanada** [comment]
I mentioned it in chat but I'm not sure you saw, you can also test most of the pipeline in this pull request by force pushing changes and checking the results on Azure, except for things that require credentials.

**Comment by joshuaboniface** [comment]
Did the non-RPi ARM fixes end up making it in with another PR (I think so?). If so, I think we can close this one out - I think consensus at this point is that I'll lose what's left of my fragile sanity if we try to switch build systems again, so GH Actions is sort of off the table.

**Pull request closed without merging**